src/sentry/incidents/subscription_processor.py

```diff
@@ -253,6 +257,40 @@ def get_aggregation_value(
 
         return aggregation_value
 
+    def handle_trigger_anomalies(
+        self,
+        has_anomaly: bool,
+        trigger: AlertRuleTrigger,
+        aggregation_value: float,
+        fired_incident_triggers: list[IncidentTrigger],
+    ) -> list[IncidentTrigger]:
+        trigger_matches_status = self.check_trigger_matches_status(trigger, TriggerStatus.ACTIVE)
+
+        if has_anomaly and not trigger_matches_status:
+            metrics.incr(
+                "incidents.alert_rules.threshold.alert",
```

Comment:
just curious - do you have any plans to make a new data dog dashboard for incidents? It could be a cool use case to have the workflow engine powerpack + all these metrics on a single dashboard.